New version in spanish

```diff
@@ -0,0 +1,142 @@
+I did a new code but i translated it to spanish and i added some color, if you accept this new version i'll get a 10 in my class an i'll love you
```

Comment:
@Frankiriva Translation looks good, but are you aware that this won't compile with this adorable comment left at the top of this file?